tests/o2a_libs/test_el_parser.py

```diff
@@ -53,9 +60,9 @@ class TestElParser(unittest.TestCase):
             ),
             ("#{customer.lName}", "{{customer.l_name}}"),
             ("#{customer.calcTotal}", "{{customer.calc_total}}"),
-            ('${wf:conf("jump.to") eq "ssh"}', '{{wf_conf("jump.to") == "ssh"}}'),
-            ('${wf:conf("jump.to") eq "parallel"}', '{{wf_conf("jump.to") == "parallel"}}'),
-            ('${wf:conf("jump.to") eq "single"}', '{{wf_conf("jump.to") == "single"}}'),
+            ('${wf:conf("jump.to") eq "ssh"}', '{{conf == "ssh"}}'),
```

Comment:
Why conf ? and what did you do to "jump.to" ?

Reply:
Fixed 🙂